src/sql/parts/jobManagement/views/jobsView.component.ts

```diff
@@ -950,7 +952,10 @@ export class JobsViewComponent extends JobManagementView implements OnInit, OnDe
 
 	public openCreateJobDialog() {
 		let ownerUri: string = this._commonService.connectionManagementService.connectionInfo.ownerUri;
-		this._commandService.executeCommand('agent.openJobDialog', ownerUri);
+		this._didDialogOpen = true;
```

Comment:
I think we shouldn't need to add a special case for the new job dialog scenario. doing the history fetching sequentially will fix the problem.